cid-redirects.json

```
@@ -1,4 +1,13 @@
 {
+    "/cid/5244": "/docs/metrics/metric-queries-alerts/operators",
+    "/cid/5245": "/docs/search/get-started-with-search/search-page/modify-search-from-messages-tab",
+    "/cid/5246": "/docs/dashboards/edit-dashboards/manage-panels",
+    "/cid/5247": "/docs/search/search-query-language/group-aggregate-operators",
+    "/cid/5248": "/docs/get-started/library/",
```

one more trailing slash

```suggestion
    "/cid/5248": "/docs/get-started/library",
```